packages/auth/src/mfa/assertions/totp.ts

```diff
@@ -155,10 +158,21 @@ export class TotpMultiFactorAssertionImpl
 
   /** @internal */
   _finalizeSignIn(
-    _auth: AuthInternal,
-    _mfaPendingCredential: string
+    auth: AuthInternal,
+    mfaPendingCredential: string
   ): Promise<FinalizeMfaResponse> {
-    throw new Error('method not implemented');
+    _assert(
+      typeof this.enrollmentId !== 'undefined' &&
```

Comment:
nit: you can just do `_assert(this.enrollment !== undefined)`

(please also update the same on line 148)

Reply:
Done